nss-day-cohort-30/bangazon-site-billowy-beluga
# Users can view product categories and the latest three products in each category

# Description
The "categories" hyperlink takes the user to a list of product categories which include the latest three products posted in each category.

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout branchdressing`
1. start BangazonSite.sln
1. click the "Categories" hyperlink in the navbar and check for functionality


# Checklist:
- [] I have commented my code, particularly in hard-to-understand areas
